src/services/events/events_test.go

```diff
@@ -2,8 +2,10 @@ package events_test
 
 import (
 	"bytes"
+	"fmt"
 	"github.com/Nastya-Kruglikova/cool_tasks/src/models"
 	"github.com/Nastya-Kruglikova/cool_tasks/src/services"
+	"github.com/satori/go.uuid"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
```

Comment:
Separate internal and external libs, please
1. "bytes",
2. cool_tasks/.. ,
3. exetrnal(satori/...)